apiServices/userServices.ts

```diff
@@ -52,3 +52,13 @@ function createUploadData(uri: string, body?: { [key: string]: string }) {
 
   return formData
 }
+
+export async function addLover(userId: string): Promise<IUser> {
+  const res = await authFetch<IUser>('POST', 'api/users/relationship', {
+    user_id: userId
+  })
+  if (res.status !== 200) {
```

Comment:
Code smell in this if statement. I recommend create two class and use polymorphism to solve it.